tla/consensus/ccfraft.tla

```diff
@@ -1044,7 +1044,8 @@ RcvAppendEntriesResponse(i, j) ==
         /\ m.type = AppendEntriesResponse
         /\ \/ HandleAppendEntriesResponse(m.dest, m.source, m)
            \/ DropResponseWhenNotInState(m.dest, m.source, m)
-           \/ DropStaleResponse(m.dest, m.source, m)
+           \/ /\ m.success
```

Comment:
```suggestion
          \* CCF does not drop NACKs under the same conditions as acks, because it re-uses the term
          \* field as an optimisation to convey the term of the last matching entry. See #5927.
           \/ /\ m.success
```